replica/database.cc

```diff
@@ -1842,19 +1842,26 @@ void dirty_memory_manager::start_reclaiming() noexcept {
 
 namespace replica {
 
-future<> database::apply_in_memory(const frozen_mutation& m, schema_ptr m_schema, db::rp_handle&& h, db::timeout_clock::time_point timeout) {
-    auto& cf = find_column_family(m.column_family_id());
+future<> database::do_apply_in_memory(const frozen_mutation& m, table& tbl, db::rp_handle&& h, db::timeout_clock::time_point timeout) {
+    data_listeners().on_write(tbl.schema(), m);
+
+    return tbl.apply(m, tbl.schema(), std::move(h), timeout);
+}
 
-    data_listeners().on_write(m_schema, m);
+future<> database::do_apply_in_memory(const mutation& m, column_family& cf, db::rp_handle&& h, db::timeout_clock::time_point timeout) {
```

Comment:
why `column_family& cf` here and `table& tbl` above?
let's seize the moment and modernize the interface.

I'd also consider reversing the order so that table& would be the first parameter and mutation/frozen_mutation the second...

Reply:
Will rename `column_family` to `table`. Already did this in some other places.